docker/release/dockerfiles/opensearch.al2.dockerfile

```diff
@@ -21,6 +21,7 @@ ARG TEMP_DIR=/tmp/opensearch
 ARG OPENSEARCH_HOME=/usr/share/opensearch
 ARG SECURITY_PLUGIN_DIR=$OPENSEARCH_HOME/plugins/opensearch-security
 ARG PERFORMANCE_ANALYZER_PLUGIN_DIR=$OPENSEARCH_HOME/plugins/opensearch-performance-analyzer
+ARG PERFORMANCE_ANALYZER_PLUGIN_NAME=opensearch-performance-analyzer
```

Comment:
Better way:
```


ARG PERFORMANCE_ANALYZER_PLUGIN_NAME=opensearch-performance-analyzer

ARG PERFORMANCE_ANALYZER_PLUGIN_DIR=$OPENSEARCH_HOME/plugins/$PERFORMANCE_ANALYZER_PLUGIN_NAME
```

Reply:
@sruti1312 I notice that `PERFORMANCE_ANALYZER_PLUGIN_DIR` is not even used anymore, if you can confirm that please remove it.

Thanks.

Reply:
It is still being used here, 
https://github.com/opensearch-project/opensearch-build/blob/main/docker/release/dockerfiles/opensearch.al2.dockerfile#L42

Reply:
That is the wrong location isnt it?

$PERFORMANCE_ANALYZER_PLUGIN_DIR/config/

It resolved to plugins/PA/config which you have already moved to config/PA

Reply:
```
if [[ -d $PERFORMANCE_ANALYZER_PLUGIN_DIR ]] ; then cp -v $TEMP_DIR/performance-analyzer.properties $OPENSEARCH_HOME/config/$PERFORMANCE_ANALYZER_PLUGIN_NAME/; fi && \
```

It checks for the ```plugins``` directory (which will be present if PA is installed), then copies into the config folder. Check can be modified to the config directory directly. 

https://github.com/sruti1312/opensearch-build/blob/feature/pa-files/docker/release/dockerfiles/opensearch.al2.dockerfile#L43